pygmt/tests/test_blockmean.py

```diff
@@ -23,19 +23,18 @@ def test_blockmean_input_dataframe():
     assert output.shape == (5849, 3)
     npt.assert_allclose(output.iloc[0], [245.888877, 29.978707, -384.0])
 
-    return output
 
-
-def test_blockmean_wrong_kind_of_input_table_matrix():
+def test_blockmean_input_table_matrix():
     """
     Run blockmean using table input that is not a pandas.DataFrame but still a
     matrix.
     """
     dataframe = load_sample_bathymetry()
```

Comment:
Please use [`pytest.fixture`](https://docs.pytest.org/en/6.2.x/fixture.html) so that the `dataframe` can be reused. 

See the `test_plot.py` for examples if it's new to you:
https://github.com/GenericMappingTools/pygmt/blob/cc26838d5183e3c71ec1374da5915983ee6ca3ae/pygmt/tests/test_plot.py#L19-L25

Reply:
Fixed in https://github.com/GenericMappingTools/pygmt/pull/1280/commits/cf7383741156a4a3664ed687b81daf1e86700a64